packages/types/lib/testLine.d.ts

```diff
@@ -158,6 +158,7 @@ export type WebPositionTarget = {
 		x: number | string,
 		y: number | string,
 	},
+	relative?: boolean,
```

Comment:
I only don't know how it will be called for WebOS, if type is `window` or `screen`

Reply:
js-api and frontend will set `type: 'window'`, for `relative position` lines, but I can add `relative` and for `screen` if you find it useful.

Reply:
Ok, the final decision if window